source/containers.tex

```diff
@@ -1274,11 +1276,12 @@
   using allocator_type = @\seebelownc{}@;
 
 private:
-  using container_node_type = @\unspecnc@;
-  using ator_traits = allocator_traits<allocator_type>;
+  using container_node_type = @\unspecnc@;                  // \expos
+  using ator_traits = allocator_traits<allocator_type>;     // \expos
```

Comment:
I think this is missing a space:
```C++
  using container_node_type = unspecified                  // \expos
  using ator_traits = allocator_traits<allocator_type>;     // \expos
```
```suggestion
  using container_node_type = @\unspecnc@;                   // \expos
  using ator_traits = allocator_traits<allocator_type>;     // \expos
```

Reply:
This is another case of #4381. Ok to resolve this since it's a general issue?

Reply:
Maybe I was wrong. I forgot the ending `;`, which would then make it aligned.